src/content/docs/plugins/submission.md

```diff
@@ -1,4 +1,115 @@
 ---
-title: Plugin Submission
-description: How to submit your plugin to the official Vencord repository
+title: Plugin Submission & Updates
+description: Vencord Plugin submission and updates guide
 ---
+
+This guide describes steps on how to release a new plugin to Vencord and to make changes to the already existing plugin. 
+
+## Initial Plugin Submission 
+
+### Pre-submission Checklist 
+
+- The folder in which stores your plugins should have camelCase name (for instance, `myAwesomePlugin`). 
+- `README.md` is in your plugin folder with a description, usage instructions, screenshots(if any) 
+- The coding of your plugin adheres to Vencord’s coding standards 
+- This is the best because probably, you have engaged in rigorous testing of the plugin. 
+
+### Submission Steps 
+
+1. **Fork the Repository**: Go to the specific repository on [https://github.com/Vendicated/Vencord](https://github.com/Vendicated/Vencord) page and select “Fork”. 
+
+2. **Clone Your Fork** 
+```bash 
+git clone https://github.com/YOUR_USERNAME/Vencord.git 
+cd Vencord 
+``` 
+
+3. **Add Your Plugin** 
+- Make sure that your plugin folder has the correct name (for example ‘myAwesomePlugin’) 
+- Next, make a copy of your plugin folder and place it in the `Vencord/src/plugins/` directory. 
+
+4. **Create a Branch** 
+```bash 
+git checkout -b add-plugin/yourPluginName 
+``` 
+
+5. **Commit Your Plugin** 
+```bash 
+git add src/plugins/yourPluginName 
+git commit -m "feat(Plugin): YourPluginName. 
+``` 
+Note: Replace ‘YourPluginName’ with your own actual plugin name 
+
+6. **Push to Your Fork** 
+```bash 
+git push –u origin add-plugin/YourPluginName 
+``` 
+
+7. **Open a Pull Request** 
+- Navigate to the fork that you created earlier on Github 
+- Click "Pull request" 
+- Set base repository to `Vendicated/Vencord` and base branch to `dev` 
+- Title your PR "feat(Plugin): YourPluginName
+- In the plugin description, the author should give a clear explanation of what the plugin does. 
+
+### Review Process 
+
+- The Vencord maintainers will simply go through your submission. 
+- Reply to any feedback as soon as possible or to changes that were requested 
+- It needs to be understood that changes might be made, thus preparing for such changes needs to be in place. 
+
+### Common Pitfalls 
+
+- Please make sure that the folder name of the plugin is the same as the one used in the commit message and the PR title 
+- Ensure that you have only committed files only found within your plugin’s directory. 
```

Comment:
There should probably be an "unless absolutely necessary" clause here

Reply:
Added :b